internal/infrastructure/middleware/enforce_identity_test.go

```diff
@@ -482,64 +506,76 @@ func TestEnforceIdentityOrder(t *testing.T) {
 	}
 }
 
+// IdentityAlwaysTrue is a predicate that always return nil
+// so everything was ok.
+// data is the reference to the identity.Identity data.
+// Return nil on success or an error with additional
+// information about the predicate failure.
+func identityAlwaysTrue(*identity.XRHID) error {
+	return nil
+}
+
+// NewIdentityAlwaysFalse is a predicate that always return
+// an error. It is useful for testing.
+func newIdentityAlwaysFalse(err error) func(*identity.XRHID) error {
```

Comment:
The comment is bit incorrect as the function itself is not a predicate. I'd use e.g. `newIdentityAlwaysFalse returns a predicate that always return specified error`

Also, I'd prefer to specify the the return value of the function as `IdentityPredicate`